sample-input/benchmarks/c5g7/c5g7-cmfd.py

```diff
@@ -1,10 +1,7 @@
-from openmoc import *
+import openmoc
 import openmoc.log as log
 import openmoc.plotter as plotter
```

Comment:
Can we remove these and just call ``openmoc.plotter``? That would allow us to just import openmoc in one line. Also it would be a bit more consistent in my opinion.